packages/launch/src/focused-launch/domain-details/test/index.tsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { MemoryRouter as Router } from 'react-router-dom';
+import { render, screen } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import DomainStep from '../';
+
+describe( 'DomainStep', () => {
+	test( 'Has title and sub-title', () => {
+		render(
+			<Router>
+				<DomainStep />
+			</Router>
+		);
+
+		expect( screen.queryByText( /Choose a domain/i ) ).toBeTruthy();
+		expect( screen.queryByText( /Free for the first year with any paid plan/i ) ).toBeTruthy();
+	} );
+
+	test( 'Has domain search input', () => {
+		render(
+			<Router>
+				<DomainStep />
+			</Router>
+		);
+
+		expect( screen.queryByPlaceholderText( /Search for a domain/i ) ).toBeTruthy();
+	} );
+
+	test( 'Has a back button', () => {
```

Comment:
Should we test for the back button to point to the correct view?

Reply:
This test case is reflected in https://github.com/Automattic/wp-calypso/issues/46943

Reply:
Cool, thank you! Not a blocker for this PR